_posts/2020-08-07-cross-origin-cookies.md

```diff
@@ -0,0 +1,49 @@
+---
+title: SameSite Cookie Policies and DHIS2 Applications
+layout: post
+categories: [blog]
+tags: [app platform, developer tools, webapp, troubleshooting, authentication]
+authors: [austin]
+---
+
+As of mid-July 2020, the Chrome stable channel [has started to disable cross-site cookies by default](https://www.chromium.org/updates/same-site). Mozilla Firefox has pushed this change to their [beta channel and will likely release it to the stable channel soon](https://www.chromium.org/updates/same-site). This change affects any DHIS2 application running **on a different domain than the DHIS2 server instance**, including applications running on localhost in development. It does not affect cross-site API requests which use Basic or OAuth authentication headers, as those do not rely on cookies for authentication.
```

Comment:
The FF link is to the chromium update page

Reply:
Nice catch, thanks!